docs/sprints/planning/SPRINT_TBD_COMPLETE_ARCHITECTURE_OVERHAUL.md

```
@@ -400,6 +413,26 @@
 - [ ] OpenTelemetry tracing active
 - [ ] Structured logging implemented
 
+### **Intelligence Integration (Throughout Sprint)**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)